mimic/apache.go

```diff
@@ -0,0 +1,320 @@
+package mimic
+
+import (
+	"bytes"
+	"crypto/rand"
+	"fmt"
+	"net"
+	"net/http"
+	"text/template"
+	"time"
+)
+
+type apacheMimic struct {
+	conn net.Conn
+	req  *http.Request
+	path string
+}
+
+var (
+	Host         string
+	Port         string
+	lastModified = time.Now().Format("Fri, 22 Oct 2015 11:52:25 GMT")
```

Comment:
Ok, find a workaround and submit an issue at https://github.com/golang/go/issues/13103